examples/ltimes.cpp

```diff
@@ -50,8 +50,8 @@
  */
 
```

Comment:
@ajkunen , I think #include<cmath> is missing in this file. I suspect that is causing ambiguity in the gcc6 build.

Reply:
that was it, thanks!